packages/database/src/migrations/20210909220543-NumberOfStudentsUpdateReportConfigs-modifies-data.js

```diff
@@ -0,0 +1,267 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const NUMBER_OF_STUDENTS_REPORT_CODES = [
+  'LESMIS_ESSDP_ece_NumberOfStudents',
+  'LESMIS_ESSDP_lse_NumberOfStudents',
+  'LESMIS_ESSDP_use_NumberOfStudents',
+  'LESMIS_ESSDP_LSE_HLO1',
+  'LESMIS_ESSDP_USE_HLO1',
+  'LESMIS_ESSDP_ECE_HLO1',
+];
```

Comment:
Since this is a large PR split between a few migrations I just reapplied the full conversion to the specific reports that were added.